src/test/java/com/epam/rp/api/tests/ApacheApiTest.java

```diff
@@ -0,0 +1,275 @@
+package com.epam.rp.api.tests;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class ApacheApiTest extends ApiBaseTest {
+
+    @Test
+    public void testCreateDashboard_ValidParameters() throws IOException {
+        HttpClient client = HttpClients.createDefault();
+        HttpPost postRequest = new HttpPost(baseUrl + "/api/v1/" + projectName + "/dashboard");
+        postRequest.setHeader("Authorization", "Bearer " + apiToken);
+        postRequest.setHeader("Content-type", "application/json");
+        String randomDescription = "Desc " + UUID.randomUUID();
+        String randomName = "Name " + UUID.randomUUID();
+        String jsonBody = String.format("{ \"description\": \"%s\", \"name\": \"%s\" }", randomDescription, randomName);
+        postRequest.setEntity(new StringEntity(jsonBody));
+        HttpResponse response = client.execute(postRequest);
```

Comment:
Looks good, I would suggest to introduce a set of utility methods for avoid code duplication:
e.g:
```java
private HttpResponse post(String url, String jsonBody) {
        HttpClient client = HttpClients.createDefault();
        HttpPost postRequest = new HttpPost(url);
        postRequest.setHeader("Authorization", "Bearer " + apiToken);
        postRequest.setHeader("Content-type", "application/json");
        postRequest.setEntity(new StringEntity(jsonBody));
        return client.execute(postRequest);
}
```

and same for all other used types of requests. I believe auth headers setting also could be unified, but the initial update should bring significant code reduction.

Reply:
Thanks for your comment, it is updated now